packages/theme-chalk/src/menu.scss

```
@@ -161,42 +157,32 @@
 
       & .#{$namespace}-menu-item.is-active,
       & .#{$namespace}-menu-item.is-active:hover,
-      & .#{$namespace}-sub-menu.is-active > .#{$namespace}-sub-menu__title,
-      &
-        .#{$namespace}-sub-menu.is-active
-        > .#{$namespace}-sub-menu__title:hover {
+      & .#{$namespace}-sub-menu.is-active>.#{$namespace}-sub-menu__title,
+      & .#{$namespace}-sub-menu.is-active>.#{$namespace}-sub-menu__title:hover {
         color: getCssVar('menu-active-color');
```

Stylelint (17.6.0)

[error] Expected "getCssVar" to be "getcssvar" (function-name-case)

(function-name-case)